static/js/global.js

```diff
@@ -1,7 +1,7 @@
 var ai = [
   { url: "http://www.ubuntu.com", title:"Ubuntu" },
```

Comment:
Yep. We should merge this first, but I've made an issue for replacing www.ubuntu.com links:
https://github.com/ubuntudesign/www.ubuntu.com/issues/907

I'll add a comment now about updating global nav.